teps/0091-trusted-resources.md

```diff
@@ -336,39 +262,28 @@ metadata:
   name: config-trusted-resources
   namespace: tekton-pipelines
 data:
-  onError: "stopAndFail"
+  skip-verification: false
   cosign-pubkey-path: "/etc/signing-secrets/cosign.pub"
   kms-pubkey-path: "gcpkms://projects/<project>/locations/<location>/keyRings/<keyring>/cryptoKeys/<key>"
 ```
-`onError: [ stopAndFail | continue ]`. (Optional, default `continue`)
-`stopAndFail` will cause the build to fail if image verification fails,  `continue` will allow the build to continue even
-if verification fails.
 
-`skip-verification`. (Optional, default `true`), if true then directly return nil, if false then do the verfication
+`skip-verification`. (Optional, default `true`), if true then directly return nil, if false then do the verification
 
 `cosign-pubkey-path`. (Optional, default `/etc/signing-secrets/cosign.pub`), it specifies the secret path to store the cosign pubkey
 
 `kms-pubkey-path`.  (Optional, default empty), it specifies the KMS reference
 
-The key path configuration can be used as default keys when no keys are founded in `keyref`.
+The key path configuration can be used as default keys when no keys are found in `keyref`.
```

Comment:
IIUC, we are allowing multiple keys to be specified, is this correct?

If so, perhaps we want a single array value rather than separate configurations for `cosign` and `kms`?

```
public-key-paths:
  - '/etc/signing-secrets/cosign.pub'
  - 'gcpkms://<key-path>'
  - 'amskms://<key-path>'
```

This admittedly makes it nominally more difficult to _augment_ the default single value by adding additional keys (you have to specify both the default and additional values), but I think it (1) makes the API more readable and (2) provides a better path for future growth of additional keys / key types.

Reply:
Originally I want to use `configmap` as a default place to store public keys, since `configmap data` is only string-string map it doesn't support array. So multiple keys will not be supported. That's why in this tep we suggest using a CRD to handle multiple keys.